src/com/magento/idea/magento2uct/ui/ConfigurationDialog.java

```diff
@@ -271,5 +295,17 @@ private void createUIComponents() {
                         new FileChooserDescriptor(false, true, false, false, false, false)
                 )
         );
+
+        additionalPath = new LabeledComponent<>();
+        additionalPath.setComponent(new TextFieldWithBrowseButton());
+        additionalPath.getComponent().addBrowseFolderListener(
+                new TextBrowseFolderListener(
+                        new FileChooserDescriptor(false, true, false, false, false, false)
+                )
+        );
+    }
+
+    private void refreshAdditionalFields(final boolean isEnabled) {
+        additionalPath.setEnabled(isEnabled);
```

Comment:
You can add `additionalPath.setVisible(isEnabled);`

Reply:
Thanks for your code review. I fixed. Please check.